components/org.wso2.carbon.identity.discovery/src/main/java/org/wso2/carbon/identity/discovery/DiscoveryConstants.java

```diff
@@ -47,6 +47,12 @@
      * MAY contain port, path, and query parameter components.
      */
     public static final String USERINFO_ENDPOINT = "Userinfo_endpoint";
+    /**
```

Comment:
Can we do a comparison with the spec and see if there are other missing ones we can support OOTB?